SSA/Projects/LLVMRiscV/Pipeline/use/ReadMe.txt

```diff
@@ -0,0 +1,24 @@
+Here we describe how to apply use our instruction selection and apply to an LLVM IR  file.
+There are several options dependeing on the desired output format.
+
+Eitherway we assume the input format to be of the following format:
+
+define <rty> @function(<ty> %i, ....) {
+    entry:
+    ....
+     ret <rty> %some
+    }
+    
+     define i32 @main(....) {
+      %out = call <rty> @function(...)
+      ret <rty>%out
```

Comment:
If this is supposed to be a code example, you could use a markdown code block.
That is, write it like so
````
```
code goes here
```
````
That just makes it a bit easier to see where the example starts and ends